static/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -253,22 +253,27 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
         }
 
         count.total += group.totals['sum(quantity)'];
-        count[outcome] += group.totals['sum(quantity)'];
+
+        if (outcome in count) {
+          count[outcome] += group.totals['sum(quantity)'];
+        } else {
```

Comment:
i don't understand this, aren't all the outcomes in count?

Reply:
(I believe the abuse outcome is unused in sentry and are recorded with an org_id of 0 so they would never show up in our queries) https://github.com/getsentry/ops/blob/999bae6f4f34d6716f50158680122e937a82d815/cookbooks/td-agent/recipes/nginx.rb

Reply:
Just wanted to make sure that any unexpected outcome would fall into `dropped`.